IdentityCore/IdentityCore.xcodeproj/project.pbxproj

```diff
@@ -7675,6 +7675,43 @@
 				);
 				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
 				MACOSX_DEPLOYMENT_TARGET = 10.12;
+				WARNING_CFLAGS = (
```

Comment:
It's better to move warning flags into .xcconfig file instead in .pbxproj file. And it may be a great amount of work need to  be done if we add all warnings together. I think it is easier to split out to single ones? Thanks!

Reply:
I'm happy to move them into an .xcconfig, if you can show me how to do that.

These warnings were already pretty much cleared because we were using them in MSAL C++. I'm not adding any of the disabled warnings, though I'd like to make it easier for that to be done in the future.

Reply:
I see. Thanks! Make sense to add the ones not disabled. So basically those warnings should be added into identitycore/xcconfig/identitycore__common.xcconfig (most of them should already be there).

Reply:
Added them to the xcconfig. Unfortunately, there's no line continuation characters in xcconfig, so we're getting some pretty gross-looking config files :/

Reply:
> It looks like these settings also get inherited and applied ADAL/MSAL. I really don't want to spend the time fixing all their warnings -- is there a way to only apply these errors to common core?

@jbzdarkid, I believe you can overwrite those settings in MSAL and broker repositories (by adding explicit settings in appropriate library specific configs). @kaisong1990, @antrix1989 would you be able to help with that?